subcommand/lifecycle-sidecar/command.go

```diff
@@ -89,19 +91,14 @@ func (c *Command) Run(args []string) int {
 	// The loop will only exit when the Pod is shut down and we receive a SIGINT.
 	for {
 		var cmd *exec.Cmd
-		if c.http.TokenFile() == "" {
-			cmd = exec.Command(c.flagConsulBinary, "services", "register", c.flagServiceConfig)
-		} else {
-			cmd = exec.Command(c.flagConsulBinary, "services", "register", c.flagServiceConfig,
-				fmt.Sprintf("-token-file=%s", c.http.TokenFile()))
-		}
+		cmd = exec.Command(c.flagConsulBinary, consulCommand...)
```

Comment:
This could be defined with `:=` and the var definition removed now that it's not needed outside of an if statement.

Reply:
Good catch!